.github/workflows/test.yml

```diff
@@ -68,7 +68,7 @@ jobs:
 
       # Must match `shard` definition in the test matrix:
       - name: Run pytest tests
-        run: PYTHONPATH='.' pytest --num-shards=4 --shard-id=${{ matrix.shard }} -n auto tests
+        run: PYTHONPATH='.' pytest --num-shards=4 --shard-id=${{ matrix.shard }} -n auto tests --durations=0
```

Comment:
```suggestion
        run: PYTHONPATH='.' pytest --num-shards=4 --shard-id=${{ matrix.shard }} -n auto tests
```

Reply:
Ups, my last commit did not make it in time.

I've added it to [the other PR](https://github.com/typeddjango/django-stubs/pull/2666) I just opened